openlibrary/templates/work_search.html

```diff
@@ -114,7 +114,7 @@
 <div id="contentHead">
     <div id="pullOut" class="sansserif searchInside">
         <p><a href="/search/inside">Search inside</a>
-        over 2 million books</p>
+        over 9 million books</p>
```

Comment:
From a content perspective, I imagine users getting confused if the results include the contents of monographs and other things they don't recognize as books.